lessons/1e_camera.md

```diff
@@ -0,0 +1,70 @@
+
+## Setting up a Camera
+
+To use the `Render` mode of rendering, we have to add a camera. This can be done easiest in the `Layout` tab. 
+Here press `Add` or `Shift+A` and add a camera.
+
+You can look through the current `Active Camera` by clicking the camera icon in the `3D viewport` panel (`Active Camera` is settable in the `Outliner` panel by clicking the green camera, if only one camera is present, this is the active camera)
+
+ <img src="../figures/view through camera.png" width="200"/>
+
+**For an easy top view:**
+In `Object Properties` set the camera `Location` to 0 in `X`/`Y` and above the volume in `Z`, and `Rotation` to 0 for all. 
+
+ <img src="../figures/camera data properties.png" width="200"/>
+
+Then in `Object Data Properties` set the camera to `Orthographic` and the `Orthographic Scale` to encompass the volume. 
+
+ <img src="../figures/orthographic camera.png" width="200"/>
+
+This is similar to a Z projection, alghough the precise type of Z projection depends on the implemented shaders/materials.
+<details><summary>Orthographic vs Perspective cameras</summary> Orthographic cameras show all objects at the same scale, in perspective cameras, objects which are far away are smaller than those nearby. Orthographic projections are standard in microscopy (for Z projections, for example). Perspective projections are default in many other cases, as this is how we view the world usually. </details>
+
+\
+**Making other angles of view:**
+Here the location and rotation depends a lot more on your data :D. You can move the camera by the input values in `Object Properties` of the camera, or by pressing `G` and moving the mouse (constrain movement along an axis with `X` `Y` `Z` keys). `Rotation` and `Scale` can be done similarly with `R` and `S`.
```

Comment:
I had very little success moving the camera around. Also still to me it's unclear what scale would do?!

Reply:
fixed by having snap to view as initial